tools/ml-dataprep/ml_dataprep/exceptions.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Exceptions raised while generating training and validation data.
+
+This module provides exceptions to be raised in specific cases of errors
+occuring during the generation of the training and validation datasets:
+    * AppException - to be raised in case of failure of creation
+    of the temporary table containg shuffled data.
+"""
+
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+
+class AppException(Exception):
```

Comment:
`AppException` is too generic. Consider giving it more specific name to hint which module it's related to. Eg. `MLDataPrepException `

Reply:
Makes sense. Changed it in 6b8a223e1719bb8184e03447477c0896649dc27b. Thank you!